### DISCUSSION TITLE
BatteryBoi Requiring Command line tools to be installed every few seconds

### DISCUSSION THREAD

**Discussion by JaiS99** [thebarbican19/BatteryBoi#54] [Q&A]
Batteryboi upon startup, opens up a system prompt, where it says that the "python3 command needs command line tools to be installed" this still occurs when I already have python3 installed, is this normal behaviour, do I need to have command line tools to be installed, even when I have python3 installed?
<img width="463" alt="Screenshot 2023-10-19 at 1 55 48 PM" src="https://github.com/thebarbican19/BatteryBoi/assets/48391316/f0f76133-6cc6-4148-ac0c-2912a61dd9d0">
<img width="573" alt="Screenshot 2023-10-19 at 2 42 52 PM" src="https://github.com/thebarbican19/BatteryBoi/assets/48391316/bed97aec-e3c8-4a1a-a1b0-30f8e86984aa">

**Comment by thebarbican19** [comment]
Hey @JaiS99. Python is used in the app as it handles some of the backend options. Python used to be installed by default on older OS versions but I have recently discovered that this is no longer the case, hence the annoying message. I am working to remove Python in the next version. 

I am converting this to a bug Issue as it needs a fix.
